### PR TITLE
Update falco helm

### LINKED ISSUE #631: Falco bintray bug
**Describe the bug**
Elastisys is using an older version of Falco susceptible to a bug. When starting Falco, it will pull a prebuilt Falco module from bintray. Due to rate limiting on [bintray's side](https://lists.cncf.io/g/cncf-falco-dev/message/190) this will fail with the following error message:
```
* Setting up /usr/src links from host
* Running falco-driver-loader with: driver=module, compile=yes, download=yes
* Unloading falco module, if present
* Trying to dkms install falco module with GCC /usr/bin/gcc
DIRECTIVE: MAKE="'/tmp/falco-dkms-make'"
* Running dkms build failed, couldn't find /var/lib/dkms/falco/2aa88dcf6243982697811df4c1b484bcbe9488a2/build/make.log (with GCC /usr/bin/gcc)
* Trying to dkms install falco module with GCC /usr/bin/gcc-8
DIRECTIVE: MAKE="'/tmp/falco-dkms-make'"
* Running dkms build failed, couldn't find /var/lib/dkms/falco/2aa88dcf6243982697811df4c1b484bcbe9488a2/build/make.log (with GCC /usr/bin/gcc-8)
* Trying to dkms install falco module with GCC /usr/bin/gcc-6
DIRECTIVE: MAKE="'/tmp/falco-dkms-make'"
* Running dkms build failed, couldn't find /var/lib/dkms/falco/2aa88dcf6243982697811df4c1b484bcbe9488a2/build/make.log (with GCC /usr/bin/gcc-6)
* Trying to dkms install falco module with GCC /usr/bin/gcc-5
DIRECTIVE: MAKE="'/tmp/falco-dkms-make'"
* Running dkms build failed, couldn't find /var/lib/dkms/falco/2aa88dcf6243982697811df4c1b484bcbe9488a2/build/make.log (with GCC /usr/bin/gcc-5)
* Trying to load a system falco driver, if present
* Trying to find locally a prebuilt falco module for kernel 5.4.149-73.259.amzn2.x86_64, if present
* Trying to download prebuilt module from https://dl.bintray.com/falcosecurity/driver/2aa88dcf6243982697811df4c1b484bcbe9488a2/falco_amazonlinux2_5.4.149-73.259.amzn2.x86_64_1.ko
curl: (22) The requested URL returned error: 403 Forbidden
Download failed, consider compiling your own falco module and loading it or getting in touch with the Falco community
```

Consider updating the falco release from 0.26.1 to 0.26.2 where this issue was [fixed](https://github.com/falcosecurity/falco/releases/tag/0.26.2). They changed the DRIVERS_REPO to their own website. There it should work.

**To Reproduce**
1. Follow perfectly the instructions from the Administrator manual.
2. After deploying everything, check the Falco logs.
It can happen that it will work fine. That's the issue, it's bintray's rate limiting. 

**Expected behavior**
Falco should run successfully without issue. 

**Version (add all relevant versions):**
 - Compliant kubernetes apps version v0.17.0 

**Additional context**
I will test the new version myself. If it works I might report here if I don't forget. Do not expect a pull request from me.

### PR DESCRIPTION
**Pull request by Pavan-Gunda**
**What this PR does / why we need it**:
This pr upgrades Falco helm chart to the newer version (1.16.0) and also upgrades falcosidekick. falcosidekick is now a dependency to the Falco helm chart as recommended by the falco community. 
**Which issue this PR fixes** *(use the format `fixes #<issue number>(, fixes #<issue_number>, ...)` to automatically close the issue when PR gets merged)*: fixes #631

**Public facing documentation PR** *(if applicable)*

**Special notes for reviewer**:

**Checklist:**

- [x] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [x] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [ ] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.
- Chart checklist (pick exactly one):
    - [ ] I upgraded no Chart.
    - [x] I upgraded a Chart and determined that no migration steps are needed.
    - [ ] I upgraded a Chart and added [migration steps](https://github.com/elastisys/compliantkubernetes-apps/blob/main/migration).

**Pipeline config** *(if applicable)*
If you change some config options (e.g. add/rename variable or change the default value) you may need to update the config used by the pipeline in `pipeline/config`.

### CODE REVIEW

File: helmfile/values/falco.yaml.gotmpl [hunk 1]
```diff
@@ -1,17 +1,14 @@
-image:
-  tag: 0.26.1
-  pullPolicy: IfNotPresent
+
+
+kubernetesSupport:
+  enableNodeFilter: true
+
```
[the remark this answers is not in the document]
Comment (Pavan-Gunda) [reply helmfile/values/falco.yaml.gotmpl]: removed

File: helmfile/values/falco.yaml.gotmpl [hunk 2]
```diff
@@ -1,17 +1,11 @@
-image:
-  tag: 0.26.1
-  pullPolicy: IfNotPresent
+
+
```
Comment (viktor-f) [inline helmfile/values/falco.yaml.gotmpl:2]: Remove these empty lines.
```suggestion
```

**Review by viktor-f — COMMENTED** [review]
Just squash the commits now and add a commit message with prefix `apps wc:`

File: helmfile/values/falco.yaml.gotmpl (hunk 2 above)
Comment (Pavan-Gunda) [reply helmfile/values/falco.yaml.gotmpl]: done!!

**Review by viktor-f — APPROVED** [review]
LGTM

**Comment by crssnd** [comment]
Did you checked if the Grafana dashboard still works after the upgrade? same for Prometheus alerts.

**Comment by Pavan-Gunda** [comment]
> Did you checked if the Grafana dashboard still works after the upgrade? same for Prometheus alerts.

yes, I did.

**Comment by crssnd** [comment]
auditLog seems to be disabled by default in falco values.yaml file.
please remove it from helmfile/values/falco.yaml.gotmpl.

File: helmfile/upstream/falco/values.yaml [hunk 3]
```diff
@@ -23,7 +44,7 @@ resources:
     cpu: 100m
     memory: 512Mi
   limits:
-    cpu: 200m
+    cpu: 1000m
```
[the remark this answers is not in the document]
Comment (Pavan-Gunda) [reply helmfile/upstream/falco/values.yaml:47]: No, should I change cpu limits back to 200m?

**Comment by Pavan-Gunda** [comment]
I have now removed audit log values @crssnd

File: helmfile/upstream/falco/values.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (crssnd) [reply helmfile/upstream/falco/values.yaml:47]: please update the wc-config.yaml with the number recommanded by the community, if it is bigger then our default

**Review by crssnd — APPROVED** [review]

**Comment by Pavan-Gunda** [comment]
Thanks for the comments. I am merging it now.

**Pull request merged**